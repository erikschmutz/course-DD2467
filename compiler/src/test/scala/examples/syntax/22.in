let f a b = a + b