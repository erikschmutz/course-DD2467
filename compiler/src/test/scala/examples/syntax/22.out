Program(
    List(
        Assignment(
            Identifier(f),
            LetBinding(
                Identifier(a),
                LetBinding(
                    Identifier(b),
                    OperatorExpr(
                        Identifier(a),
                        Identifier(b),
                        Plus()
                    )
                )
            )
        )
    )
)